sdk/ml/azure-ai-ml/azure/ai/ml/_restclient/v2023_04_01_preview/models/_models_py3.py

```diff
@@ -28138,6 +28138,8 @@ class SweepJob(JobBaseProperties):
     :vartype search_space: any
     :ivar trial: Required. [Required] Trial component definition.
     :vartype trial: ~azure.mgmt.machinelearningservices.models.TrialComponent
+    :ivar resources: Compute Resource configuration for the job.
```

Comment:
Please donot change anything under _restclient, this is autogenerated..

Reply:
Thanks, removed